classes/Integration/Form/HappyForms.php

```
@@ -0,0 +1,176 @@
+		}
+
+		return $form_selectlist;
+	}
+
+	/**
+	 * Hooks in a success function specific to this provider for non-AJAX submission handling.
+	 *
+	 * @param array $submission The submission data.
+	 * @param array $form       The form data.
+	 */
+	public function on_success( $submission, $form ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$submission'. (undefined)

(UnusedFormalParameter)